Include/internal/pycore_object.h

```diff
@@ -161,6 +209,36 @@ _Py_DECREF_NO_DEALLOC(PyObject *op)
 #endif
 }
 
+#else
+static inline void
+_Py_DECREF_SPECIALIZED(PyObject *op, const destructor destruct)
+{
+    Py_DECREF(op);
+}
+
+static inline void
+_Py_DECREF_NO_DEALLOC(PyObject *op)
```

Comment:
Given we know that the refcount cannot reach zero, why not take advantage of that?